webpack.config.js

```diff
@@ -1,30 +1,35 @@
-/**
- * This file is used by all packages/wonder-blocks-x/webpack.config.js files.
- *
- * It generates an appropriate webpack configuration for each package.  In
- * particular it adds all dependencies from the specific package along with
- * project level dependencies to the list of externals.  This causes webpack
- * to not include those depedencies in the build file it generates.
- */
+const fs = require("fs");
 const path = require("path");
 
-module.exports = function(subPkgRoot) {
-    const subPkgJson = require(path.join(subPkgRoot, "./package.json"));
+const packages = fs
```

Comment:
🛑 Would be better to rewrite this as:

```
const pkgsDir = path.join(__dirname, "packages");
const packages = fs
	.readdirSync(pkgsDir)
	.map((dir) => path.join(pkgsDir, dir));
```

(best to not use string concatenation when building file paths)